### PR TITLE
Benchmark report consistency fix

### PR DESCRIPTION
**Pull request by yurloc**
Score "calculation" count summary is used in the navigation button and in documentation.

### CODE REVIEW

File: optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/report/BenchmarkReport.java [hunk 1]
```diff
@@ -498,8 +498,8 @@ private void writeAverageCalculateCountPerSecondSummaryChart() {
         }
         XYPlot plot = createScalabilityPlot(seriesList,
                 "Problem scale", NumberFormat.getInstance(locale),
-                "Average calculate count per second", NumberFormat.getInstance(locale));
-        JFreeChart chart = new JFreeChart("Average calculate count summary (higher is better)",
+                "Average calculation count per second", NumberFormat.getInstance(locale));
+        JFreeChart chart = new JFreeChart("Average calculation count summary (higher is better)",
```
Comment (ge0ffrey) [inline optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/report/BenchmarkReport.java]: Some time ago, I renamed "average calculation count per second (999)" to "score calculation speed (999/sec)". Looks like I missed these - we should rename them also.

Comment (yurloc) [inline optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/report/BenchmarkReport.java]: Ahh, sorry, I forgot to include an important notice: this is for 6.x stream, the fix doesn't apply to master.

**Comment by yurloc** [comment]
There are other occurrences of the "calculate count" phrase but I only chose to fix this one because documentation uses "calculation count" when describing this part of the benchmark report (see 14.4.6. Average Calculation Count Summary (Graph and Table)).

File: optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/report/BenchmarkReport.java (hunk 1 above)
Comment (ge0ffrey) [inline optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/report/BenchmarkReport.java]: ah, I hadn't notice that, merging, thanks :)

**Pull request merged**